index.php

```diff
@@ -0,0 +1,51 @@
+<?php
+?>
+
+<!DOCTYPE HTML>
+<HTML>
+    <head>
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+        <link rel="stylesheet" type="text/css" href="styles/styles.css" >
```

Comment:
left a space before closing the link tag

Reply:
Changed!